tests/libtransmission/json-test.cc

```diff
@@ -127,32 +118,19 @@ TEST_P(JSONTest, testUtf8)
     var.clear();
 
     EXPECT_FALSE(std::empty(json));
-    EXPECT_NE(std::string::npos, json.find("\\u00f6"));
-    EXPECT_NE(std::string::npos, json.find("\\u00e9"));
+    EXPECT_EQ(R"({"key":"Letöltések"})"sv, json);
     var = serde.parse(json).value_or(tr_variant{});
     EXPECT_TRUE(var.holds_alternative<tr_variant::Map>());
     EXPECT_TRUE(tr_variantDictFindStrView(&var, key, &sv));
     EXPECT_EQ("Letöltések"sv, sv);
-}
-
-TEST_P(JSONTest, testUtf16Surrogates)
-{
-    static auto constexpr ThinkingFaceEmojiUtf8 = "\xf0\x9f\xa4\x94"sv;
```

Comment:
Shouldn't we keep three lines of test for `"\xf0\x9f\xa4\x94"sv`? Just to assert that we're not having regressions here:
```
     tr_variantDictAddStr(&var, key, "\xf0\x9f\xa4\x94"sv);
     json = serde.to_string(var);
     EXPECT_EQ(R"({"key":"SOMETHING..."})"sv, json);
```

Reply:
I can incorporate this string in `testUtf8` too, sure.

Reply:
7ea2fac